app/views/noisy_workflow/skip_review.text.erb

```diff
@@ -0,0 +1,10 @@
+Hello,
+
+Review has been skipped for "<%= @edition.title %>" (<%= @edition.format %>).
+
+You can view it here:
+<%= "#{edition_url(@edition.id)}" %>
+
+Regards,
+
+Winston (the GOV.UK Publisher)
```

Comment:
I'm still trying to work out who Winston is...

Reply:
I have no idea! All the emails seem to be from Winston? Not sure if this is only for internal emails?